_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -944,14 +936,66 @@ https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer re
 https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin[vulnerabilities with the cellular system],
 so using a virtual or hardware MFA device is preferable; that said, MFA with SMS is still better than no MFA at all.
 
+[[deployment_approach]]
+=== Deployment approach
+Before we dive into the code and deployment for each of the resources, let's take a step back and understand how the code is structured.
+Most of the features explained in the <<production_grade_design>> section will be deployed by using the Landing Zone solution, and some more standalone modules like the VPC module.
+
+The Landing Zone will be deployed in three steps - the `account-baseline-root` to set up your organisation-wide configurations, create the necessary child AWS accounts, set up the CloudTrail and AWS Config buckets. Next, we'll need to apply the `account-baseline-app` against the created logs account, adding more settings that will be used for aggregation of logs and metrics from the whole organization. Then the `account-baseline-security` will be applied, and that's responsible to set up your IAM roles, groups that would allow you to access the rest of the accounts within your organization. And finally, the `account-baseline-app` will be applied to an AWS account with the purpose of hosting an application.
```

Comment:
```suggestion
The Landing Zone will be deployed in three steps - the `account-baseline-root` to set up your organization-wide configurations, create the necessary child AWS accounts, set up the CloudTrail and AWS Config buckets. Next, we'll need to apply the `account-baseline-app` against the created logs account, adding more settings that will be used for aggregation of logs and metrics from the whole organization. Then the `account-baseline-security` will be applied, and that's responsible to set up your IAM roles and groups that would allow you to access the rest of the accounts within your organization. And finally, the `account-baseline-app` will be applied to an AWS account with the purpose of hosting an application.
```